source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -40,12 +40,40 @@ Configure Prometheus to Collect and Alert using MinIO Metrics
 
 Use the :mc-cmd:`mc admin prometheus generate` command to generate the scrape configuration for use by Prometheus in making scraping requests:
 
-.. code-block:: shell
-   :class: copyable
+.. tab-set::
+
+   .. tab-item:: MinIO Server
+
+      The following command scrapes metrics for the MinIO Server.
+
+      .. code-block:: shell
+         :class: copyable
+      
+         mc admin prometheus generate ALIAS
+
+      Replace :mc-cmd:`ALIAS <mc admin prometheus generate TARGET>` with the :mc:`alias <mc alias>` of the MinIO deployment.
+
+   .. tab-item:: Nodes
+
+      The following command scrapes metrics for the MinIO Server.
```

Comment:
this should only scrape metrics for a single MinIO node - though I'm not sure *which* node. Or maybe it's general node-level metrics.

Reply:
Good catch